.gitignore

```diff
@@ -1 +1,6 @@
-*.pyc
\ No newline at end of file
+<<<<<<< HEAD
+*.pyc
+.vscode/
+=======
+*.pyc
+>>>>>>> bfb91dc323d330bd464faf1104c8563d8a6ae545
```

Comment:
Why?

Reply:
Nothing I intentionally added. Probably copy pasted it in accidently